nettacker/core/utils/dsl_matcher.py

```
@@ -0,0 +1,494 @@
+        if not content or not patterns:
+            return None
+        patterns = [patterns]
+        for pattern in patterns:
+            try:
+                match = re.search(pattern, content, re.IGNORECASE)
+                if match:
+                    if match.groups():
+                        return match.group(1)
+                    return match.group(0)
+            except Exception:
+                continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)